src/components/SearchBulkOperationsPanel/index.js

```diff
@@ -58,7 +54,7 @@ export default class SearchBulkOperationsPanel extends PureComponent {
                     , false
                   )
                 }
-                onClick={this.handleDelete}
+                onClick={this.handle(deleteInstances, selectedInstances)}
```

Comment:
This is a React.js anti-pattern since new handler is generated on every **render()** call.